packages/form/package.json

```diff
@@ -56,8 +56,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@astro-reactive/common": "^0.1.2",
-    "@astro-reactive/validator": "^0.2.3",
+    "@astro-reactive/common": "*",
+    "@astro-reactive/validator": "*",
```

Comment:
Question! :) What do the asterisks mean here? Will they be updated automatically by turborepo?

When the package is released to npm, these need to be set exactly in the latest versions because this `package.json` file will be released in the package 🤔

Reply:
the asterisk should be resolved by turbo for sharing internal packages.
 > When the package is released to npm, these need to be set exactly in the latest versions because this package.json file will be released in the package 

Yep, I still also have some doubt about how turbo will resolve packages that will be published. But, what I found in the wild, such as Astro's repo, they use `*` too when they use their published packages. I will look into it further.

Maybe we can test it in experimental release later?